decentraland/helpers.js

```diff
@@ -1,9 +1,158 @@
-const { Action } = require("tasit-sdk");
+import { SecureStore } from "expo";
+import { Toast } from "native-base";
+import {
+  Account,
+  Action,
+  ContractBasedAccount,
+  TasitContracts,
+} from "tasit-sdk";
 const { ConfigLoader } = Action;
-const { ProviderFactory } = require("tasit-action/dist/ProviderFactory");
-const tasitSdkConfig = require("./config/default.js");
+import ProviderFactory from "tasit-action/dist/ProviderFactory";
```

Comment:
It's interesting that the switch to remove `require` and the switch to remove the `{ }` coincided. Is that expected? I'm sure you've tried it, so probably.

Reply:
Yes, it is. I've made this sometimes before:
`const var = require("mod").var` but `const { var } = require("mod")` is better.

Reply:
Hmm, but here there's no `.ProviderFactory` in one version that would make this like your example, is there?

Reply:
Oh, you're right.

`const { ProviderFactory } = require("tasit-action/dist/ProviderFactory");`
console.log(ProviderFactory)
// undefined

Weird... how it was working?!

Reply:
Hmm, interesting